packages/react/src/blocks/Text.tsx

```diff
@@ -25,9 +25,28 @@ class TextComponent extends React.Component<TextProps> {
   }
 
   evalExpression(expression: string, state: any) {
+    // Don't interpolate when inline editing
+    if (this.allowTextEdit) {
+      return String(expression);
+    }
     return String(expression).replace(/{{([^}]+)}}/g, (match, group) => tryEval(group, state));
   }
 
+  get allowTextEdit() {
+    return (
+      Builder.isBrowser &&
+      Builder.isEditing &&
+      location.search.includes('builder.allowTextEdit=true') &&
+      !(
+        this.props.builderBlock &&
+        this.props.builderBlock.bindings &&
+        (this.props.builderBlock.bindings['component.options.text'] ||
+          this.props.builderBlock.bindings['options.text'] ||
+          this.props.builderBlock.bindings['text'])
+      )
+    );
+  }
```

Comment:
Would this change need to be ported to the new SDKs too? What is this text interpolation exactly?

Reply:
Cool. I actually see that interpolation isn't supported to begin with in the new SDKs yet